src/main/java/mate/academy/service/CinemaHallService.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.service;
+
+import java.util.List;
+import mate.academy.model.CinemaHall;
+
+public interface CinemaHallService {
+    CinemaHall add(CinemaHall cinemaHall);
+
+    CinemaHall get(Long id);
```

Comment:
According to the checklist, let's return `Optional<CinemaHall>` in `get()` method on the service layer, and the entity itself on the dao layer.